src/backend/app/tasks/tasks_crud.py

```diff
@@ -330,22 +328,20 @@ async def update_task_history(
 
 
 async def get_project_task_history(
-    project_id: int,
+    task_id: int,
     comment: bool,
     end_date: datetime,
-    task_id: Optional[int],
     db: Session,
 ):
     """Retrieves the task history records for a specific project.
 
     Args:
-        project_id (int): The ID of the project.
+        task_id (int): The task_id of the project.
```

Comment:
Related to https://github.com/hotosm/fmtm/issues/1326

Is it possible to keep both `project_id` and `task_id` params here, as in the future we want to be using the task id in relation to the project?

i.e. instead of `task_id=13879435` we want to use `project_id=22&task_id=10`

If needed for now we could pass a combination: `project_id=22&task_id=13879435`, where only the task id is used in the SQL. But we may refactor to use the project_id/task_id combo in the future & keeping both would make that easier.

Reply:
Yes, great, Thanks for pointing it out.